src/main/java/AddTaskCommand.java

```diff
@@ -0,0 +1,50 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+/**
+ * Generates the correct type of task and adds it to the task list
+ */
+
+public class AddTaskCommand extends Command {
```

Comment:
I believe class names should be nouns written in camelCasing according to coding standards.